master/static/srv/shell-entrypoint.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 "$DET_PYTHON_EXECUTABLE" -m pip install -q --user /opt/determined/wheels/determined*.whl
 
-pushd ${WORKING_DIR} && test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}" && popd
+cd ${WORKING_DIR} && test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
```

Comment:
remove `cd ${WORKING_DIR} &&`, as I mentioned in my comment.

Also remove the WORKING_DIR variable entirely while you are at it.

Reply:
Done!